src/pybullet_helpers/robots/fetch.py

```diff
@@ -0,0 +1,40 @@
+"""Fetch Robotics Mobile Manipulator (Fetch)"""
+from predicators.src import utils
+from predicators.src.pybullet_helpers.robots.single_arm import \
+    SingleArmPyBulletRobot
+
+
+class FetchPyBulletRobot(SingleArmPyBulletRobot):
```

Comment:
can you make sure that we're consistent about `PyBullet` vs `Pybullet`? (find and replace `Pybullet` with `PyBullet` throughout the code base)

here's my bash function for find and replace in case it helps

```
# find and replace for git
git-replace(){
    git grep -l "$1" | xargs sed -i '' -e "s/$1/$2/g"
}
```

so `git-replace Pybullet PyBullet`